upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -1413,6 +1413,9 @@ There you can find links to upgrade notes for other versions too.
 - remove subscription newsletter form from error pages ([#1819](https://github.com/shopsys/shopsys/pull/1819))
     - see #project-base-diff to update your project
 
+- non-existent product returns 410 ([#1829](https://github.com/shopsys/shopsys/pull/1829))
```

Comment:
this sentence should be an instruction. Something like `split error page templates to allow to render new 410 error response`

Reply:
d3211f0e5b4719d61bc72772f710c0884d07efbb, a0d166fc95fa565e905106f4be36f928a43fb5ce